tests/test_hodgkin.py

```diff
@@ -307,3 +307,10 @@ def test_get_equations_for(self, graph, model):
         # Check not lexicographical sorted equations against expected equations
         for i in range(len(unordered_equations)):
             assert str(unordered_equations[i]) == str(unordered_ref_eq[i])
+
+    def test_dimensionally_equivalent(self, model):
```

Comment:
This doesn't go here! How would I ever know that `test_hodgkin` was the place to look for a test of a method in the UnitStore class?

There should be a test_unit_store somewhere, or perhaps a test_units, where this will fit much better

Reply:
Made the requested change.
However if you do have a point but if you follow that through the following tests shouldn't be there either as they are testing the respective functions and not the model.

`test_get_symbol_by_ontology_term`
`test_get_ontology_terms_by_symbol`
`test_has_ontology_annotation`
`test_get_equations_for`

Of course that would need a separate pull request!

Reply:
Good point!
Thanks for moving it